.github/workflows/lints.yml

```diff
@@ -8,7 +8,9 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
```

Comment:
Any reason not to bump this to @v3 given the other changes you're making? 
https://github.com/actions/checkout

Reply:
Good call!